recipes/noman

```diff
@@ -0,0 +1 @@
+(noman :repo "andykuszyk/noman.el" :fetcher github :files ("noman.el"))
```

Comment:
This recipe is at least currently equivalent, and MELPA prefers the default recipe when possible:
```suggestion
(noman :repo "andykuszyk/noman.el" :fetcher github)`
```
```

Reply:
Thanks @riscy 🙏 

I've made that change [here](https://github.com/melpa/melpa/pull/9017/commits/ea52325ba260e879a7e1a363833b6092622aa312), because I think there might have been an erroneous back tick in your suggestion. Please let me know if I'm mistaken 🙂